source/decoder.py

```diff
@@ -187,7 +187,7 @@ def get_instruments_tuple(events):
 
 if __name__ == "__main__":
 
-    path_filename = "midi/generated/misnaej/the-jam-machine/20221207_005555"
+    path_filename = "midi/generated/misnaej/the-jam-machine/20221207_200231"
```

Comment:
agreed - but I don't know how to do it. Could you quickly implement it ?

Reply:
it would not be quickly unfortunately, I'm focusing on UI now, maybe we will just use all these inside the UI and won't need any command arguments? so I'd say fuck it for now:D i just wanted to mention is all